ci-operator/step-registry/openshift-extended/rosacli-tests/openshift-extended-rosacli-tests-commands.sh

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+export HOME=/tmp/home
+export REPORT_HANDLE_PATH="/usr/bin"
+export ENABLE_PRINT_EVENT_STDOUT=true
+
+CLOUD_PROVIDER_REGION=${LEASED_RESOURCE}
+
+# Get cluster 
+CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
+echo "CLUSTER_ID is $CLUSTER_ID"
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# prepare for the future usage on the kubeconfig generation of different workflow
+test -n "${KUBECONFIG:-}" && echo "${KUBECONFIG}" || echo "no KUBECONFIG is defined"
+test -f "${KUBECONFIG}" && (ls -l "${KUBECONFIG}" || true) || echo "kubeconfig file does not exist"
+ls -l ${SHARED_DIR}/kubeconfig || echo "no kubeconfig in shared_dir"
+ls -l ${SHARED_DIR}/kubeadmin-password && echo "kubeadmin passwd exists" || echo "no kubeadmin passwd in shared_dir"
+
+which extended-platform-tests
+
+#set env for rosacli login
+if test -f "${CLUSTER_PROFILE_DIR}/ocm-token"
+then
+    ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token") || true
```

Comment:
@yuwang-RH 
```shell
    ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token") || true
    export ROSA_TOKEN
```

if `cat "${CLUSTER_PROFILE_DIR}/ocm-token"`, token is not got. you still need to export it?

Reply:
BTW, it is duplicate logic here: `test -f "${CLUSTER_PROFILE_DIR}/ocm-token` and `|| true`.